packages/frontend/src/js/app/router.js

```diff
@@ -199,7 +204,7 @@ const router = new VueRouter({
     routes: [
         {
             path: "/",
-            redirect: home,
+            beforeEnter: home,
```

Comment:
Obligé de remplacer le "redirect" par un "beforeEnter" sinon le "beforeeach" guard n'était pas exécuté.
Cf la doc : `Note that Navigation Guards are not applied on the route that redirects, only on its target.`